### PR TITLE
Peribolos: update repo permissions for GH teams

### PR DESCRIPTION
**Pull request by Priyankasaggu11929**
The PR: 

- update `repos:` permissions for the following GitHub teams under kubernetes and kubernetes-sigs orgs:
    - `cel-admission-webhook-admins`
    - `cel-admission-webhook-maintainers`
    - ` maintainers-maintainers`ask
    - `sigs-github-actions-admins`
    - `sigs-github-actions-maintainers`
    - `kube-scheduler-wasm-extension-admins`
    - `cel-admission-webhook-admins`
    - `cel-admission-webhook-maintainers`
    - `sig-security-admins`
    - `sig-security-maintainers`
    
- update `restrictions.yaml` to whitelist new repo permissions added as part of ^

- add `aroradaman` to `sig-windows-dev-tools-admins` GH team
   (as per the ask in [4370#issuecomment-1671371021](https://github.com/kubernetes/org/pull/4370#issuecomment-1671371021))

/area github-repo
/assign @MadhavJivrajani

### CODE REVIEW

**Comment by Priyankasaggu11929** [comment]
Hello @jayunit100 – could you please confirm whether [sig-windows-dev-tools-admins](https://github.com/kubernetes/org/blob/a7ec3121cce710d822e329365270dd5b942e4437/config/kubernetes-sigs/sig-windows/teams.yaml#L46-L57) GH team requires additional `admin` access for the [k-sigs/windows-service-proxy](https://github.com/kubernetes-sigs/windows-service-proxy) repository (besides, the existing `admin` access to [k-sigs/sig-windows-dev-tools](https://github.com/kubernetes-sigs/sig-windows-dev-tools) repo)?

If so, I'll update the teams config accordingly to reflect the additional repo permissions. Thanks!

**Review by MadhavJivrajani — COMMENTED** [review]
Just one comment apart from the last bit of inconsistencies being discussed being on slack: 
https://kubernetes.slack.com/archives/C01672LSZL0/p1691582605843739

**Comment by Priyankasaggu11929** [comment]
Hello @jacobwolfaws – could you please confirm if the current config mapping for **[aws-fsx-csi-driver-admins](https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-aws/teams.yaml#L55-L66)**  and **[aws-fsx-csi-driver-maintainers](https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-aws/teams.yaml#L67-L79)** GH teams, granting `admin` access to **[k-sigs/aws-fsx-csi-driver](https://github.com/kubernetes-sigs/aws-fsx-csi-driver)** repo is correct?

**_OR_** the ^ teams repo permissions needs to be updated to – `admin` access to **[k-sigs/aws-fsx-openzfs-csi-driver](https://github.com/kubernetes-sigs/aws-fsx-openzfs-csi-driver)** repo? Thanks!

**Comment by jayunit100** [comment]
yes , can we also add @aroradaman to the windows-service-proxy owners team thanks !

**Comment by Priyankasaggu11929** [comment]
Hello @logicalhan – could you please confirm if the current repo permissions mapping for **[instrumentation-admins](https://github.com/kubernetes/org/blob/de341ff702383b457c11ce94df3c85befb0bd4ed/config/kubernetes-sigs/sig-instrumentation/teams.yaml#L16-L25)** GH team, granting `admin` access to **[k-sigs/instrumentation](https://github.com/kubernetes-sigs/instrumentation)** repo is correct?

**_OR_** the team repo permissions needs to be updated to – `admin` access to **[k-sigs/intrumentation-addons](https://github.com/kubernetes-sigs/instrumentation-addons)** repo? Thanks!

**Comment by logicalhan** [comment]
> Hello @logicalhan – could you please confirm if the current repo permissions mapping for **[instrumentation-admins](https://github.com/kubernetes/org/blob/de341ff702383b457c11ce94df3c85befb0bd4ed/config/kubernetes-sigs/sig-instrumentation/teams.yaml#L16-L25)** GH team, granting `admin` access to **[k-sigs/instrumentation](https://github.com/kubernetes-sigs/instrumentation)** repo is correct?

Yes, it looks okay to me. Thanks!

File: config/kubernetes-sigs/sig-contributor-experience/teams.yaml
Comment (Priyankasaggu11929) [reply config/kubernetes-sigs/sig-contributor-experience/teams.yaml]: @MadhavJivrajani, updated in the latest commit refresh. Thanks!

**Comment by jacobwolfaws** [comment]
> Hello @jacobwolfaws – could you please confirm if the current repo permissions mapping for **[aws-fsx-csi-driver-admins](https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-aws/teams.yaml#L55-L66)** and **[aws-fsx-csi-driver-maintainers](https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-aws/teams.yaml#L67-L79)** GH teams, granting `admin` access to **[k-sigs/aws-fsx-csi-driver](https://github.com/kubernetes-sigs/aws-fsx-csi-driver)** repo is correct?
> 
> **_OR_** the ^ teams repo permissions needs to be updated to – `admin` access to **[k-sigs/aws-fsx-openzfs-csi-driver](https://github.com/kubernetes-sigs/aws-fsx-openzfs-csi-driver)** repo? Thanks!



> Hello @jacobwolfaws – could you please confirm if the current repo permissions mapping for **[aws-fsx-csi-driver-admins](https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-aws/teams.yaml#L55-L66)** and **[aws-fsx-csi-driver-maintainers](https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-aws/teams.yaml#L67-L79)** GH teams, granting `admin` access to **[k-sigs/aws-fsx-csi-driver](https://github.com/kubernetes-sigs/aws-fsx-csi-driver)** repo is correct?
> 
> **_OR_** the ^ teams repo permissions needs to be updated to – `admin` access to **[k-sigs/aws-fsx-openzfs-csi-driver](https://github.com/kubernetes-sigs/aws-fsx-openzfs-csi-driver)** repo? Thanks!

Those look right to me, openzfs has separate permissions defined in another file.

**Comment by Priyankasaggu11929** [comment]
Thanks for the confirmation @jacobwolfaws.

@MadhavJivrajani, the PR is ready for another round of review. Thanks!

File: config/kubernetes-sigs/sig-windows/teams.yaml
Comment (MadhavJivrajani) [inline config/kubernetes-sigs/sig-windows/teams.yaml:1]: Leaving breadcrumbs in case anyone wants to see why this addition was done:
Reason for addition here: https://github.com/kubernetes/org/pull/4370#issuecomment-1671371021

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]
@Priyankasaggu11929 a few more inconsistencies:
- `kubernetes/sig-architecture-leads` needs **admin** access to repo `kubernetes/design-proposals-archive` (ref https://github.com/kubernetes/community/issues/6055)
- `kubernetes/registry.k8s.io-admins` needs  **admin** access to repo `kubernetes/registry.k8s.io`
- `kubernetes/registry.k8s.io-maintainers` needs **write** access to repo `kubernetes/registry.k8s.io`

From the list share don slack: https://kubernetes.slack.com/archives/C01672LSZL0/p1691583200807829?thread_ts=1691582605.843739&cid=C01672LSZL0

**Comment by Priyankasaggu11929** [comment]
In response to https://github.com/kubernetes/org/pull/4370#pullrequestreview-1571736892:

@MadhavJivrajani – added the missing repo perms  in the latest 2 commits. PTAL again. Thanks! :pray:

**Review by MadhavJivrajani — APPROVED** [review]
/lgtm 
/approve

**Review by MadhavJivrajani — APPROVED** [review]
/lgtm
/hold cancel

**Pull request merged**
